legistar/events.py

```diff
@@ -222,12 +222,14 @@ def api_events(self, since_datetime=None):
             # corresponding event modification. Query all update fields so later
             # changes are always caught by our scraper, particularly when
             # scraping narrower windows of time.
-            update_fields = ('EventLastModifiedUtc',
+            update_fields = ('EventDate',
+                             'EventLastModifiedUtc',
                              'EventAgendaLastPublishedUTC',
                              'EventMinutesLastPublishedUTC')
 
             since_fmt = " gt datetime'{}'".format(since_datetime.isoformat())
-            since_filter = ' or '.join(field + since_fmt for field in update_fields)
+            since_filter = ' or '.join(field + since_fmt
```

Comment:
Ditto comment on readability, above.

Reply:
okay, would you mind making the suggested change.

Reply:
i can do it.